docker/Dockerfile

```diff
@@ -137,7 +137,7 @@ RUN apt-get update && \
 # https://python-poetry.org/docs/master/#installing-with-the-official-installer
 # This also makes it so that Poetry will *not* be included in the "final" image since it's not installed to /usr/local/
 RUN curl -sSL https://install.python-poetry.org -o /tmp/install-poetry.py && \
-    python /tmp/install-poetry.py && \
+    python /tmp/install-poetry.py --version 1.2.0 && \
```

Comment:
Just double checking on the comment above. The installer _should_ install it in a place other than `/usr/local` but should we enforce this through providing `--path` or `$POETRY_HOME`?

Reply:
--path is for specifying where to find the file to install from, if I'm reading the script right. We could explicitly set POETRY_HOME I suppose but I feel like this is a pretty low risk.